aura: add page (#6533)
tldr-pages/tldr: pages/linux/aura.md

# aura

> The Aura Package Manager: A secure, multilingual package manager for Arch Linux and the AUR.
> More information: <https://github.com/fosskers/aura>.

> - Search for packages from the repos and AUR:

Reviewer: I can't find the `--both` flag on the GitHub page, but I assume it enables the official repos too, right?

Author: Yes all the repos in /etc/pacman.conf
And it's in the man page.

Reviewer 2: ```suggestion
- Search for packages from the official repositories and AUR:
```

> `aura -As --both {{package_name|search_regex}}`

Reviewer 2: What kind of regex this supports?

- Update all AUR packages in a verbose mode and remove all make dependencies:

`aura -Akuax`

- Install a package from the AUR:

`aura -A {{package_name}}`

> - Install a package from the system repos:

Reviewer 2: ```suggestion
- Install a package from the official repositories:
```

`aura -S {{package_name}}`

> - Update all system repo packages:

Reviewer 2: ```suggestion
- Synchronize and update all packages from the official repositories:
```

`aura -Syu`

> - Downgrade a package from the package cache:

Reviewer 2: Is `using` better? :thinking: 
```suggestion
- Downgrade a package using the package cache:
```

Author: Yea that makes sense, i'll commit that.

`aura -C {{package_name}}`

> - Remove an installed package and both its dependencies:

Reviewer: ```suggestion
- Remove both an installed package and its dependencies:
```

> - Remove both an installed package and its dependencies:

Reviewer 2: ```suggestion
- Remove a package and its dependencies:
```

`aura -Rs {{package_name}}`

> - Uninstall all orphan packages:

Reviewer 2: ```suggestion
- Remove orphan packages (installed as dependencies but not required by any package):
```

> `aura -Rsu {{package_name}}`

Reviewer 2: ```suggestion
`aura --remove --recursive --unneeded {{package_name}}`
```
